packages/core/src/components/autosuggest/autosuggest.scss

```
@@ -45,6 +45,7 @@
   .group:not(:first-child) {
     .group-label {
       border-block-start: 1px solid colors.$grijs-30;
+      padding-block-start: 3 * core-autosuggest-variables.$list-item-padding-block;
```

```suggestion
      padding-block-start: units.$padding-base-inline;
```

Nu moet ik een paar keer klikken om de padding te achterhalen. Zo is het met 1 klik te zien